firewall-use-system-defaults-ignore-options.ks.in

```diff
@@ -0,0 +1,57 @@
+
+#version=DEVEL
+#test name: firewall-use-system-defaults-ignore-options
+#
+# Test that firewall can be properly configured to use
+# system defaults. This basically means avoiding any
+# firewall-offline-cmd --enable/--disable calls and
+# simply leaving default distro provided firewall
+# configuration in place.
+%ksappend repos/default.ks
+
+%ksappend common/common_no_payload.ks
+%ksappend payload/default_packages.ks
+
+
+# Check no options are set to the target system when the
+# --use-system-defaults option is in place.
+firewall --use-system-defaults --port=22001:tcp,6400:udp --service=tftp,smtp
+
+%packages
+%end
+
+%post
+
+## TEST PROCEDURE
+# Test for 22001/TCP
+firewall-offline-cmd --list-ports | grep 22001/tcp
+if [[ $? -eq 0 ]]; then
+    echo "*** Firewall should not be configured config for 22001/tcp" >> /root/RESULT
+fi
+
+# Test for 6400/UDP
+firewall-offline-cmd --list-ports | grep 6400/udp
+if [[ $? -eq 0 ]]; then
+    echo "*** Firewall should not be configured config for 6400/udp" >> /root/RESULT
```

Comment:
Please changed this message ``configured config`` is not really meaningful.
What about ports. Something like: ``Defined firewall port 6400/udp is not opened"

Reply:
That's indeed a typo, changing. :)